replication-api/src/main/java/com/connexta/replication/api/impl/persistence/SiteManager.java

```diff
@@ -11,19 +11,19 @@
  * License is distributed along with this program and can be found at
  * <http://www.gnu.org/licenses/lgpl.html>.
  */
-package com.connexta.ion.replication.api.persistence;
+package com.connexta.replication.api.impl.persistence;
 
-import com.connexta.ion.replication.api.data.ReplicationSite;
+import com.connexta.replication.api.data.Site;
 
 /** A SiteManager performs CRUD operations for replication sites. */
-public interface SiteManager extends DataManager<ReplicationSite> {
+public interface SiteManager extends DataManager<Site> {
 
   /**
-   * Creates a new {@link ReplicationSite} implementation with the given name and url
+   * Creates a new {@link Site} implementation with the given name and url
    *
-   * @param name the name to give the {@link ReplicationSite}
-   * @param url the for the {@link ReplicationSite}
-   * @return a new {@link ReplicationSite} implementation
+   * @param name the name to give the {@link Site}
+   * @param url the for the {@link Site}
+   * @return a new {@link Site} implementation
    */
-  ReplicationSite createSite(String name, String url);
+  Site create(String name, String url);
```

Comment:
✏️ I don't think we are using this if we want to delete it.

Reply:
They are currently in the `DataManager` interface so I kept them for the time being.